docs/modules/ROOT/pages/lmeval-oauth-authentication.adoc

```diff
@@ -0,0 +1,274 @@
+      value: granite
+    - name: base_url
+      value: $ROUTE/v1/completions  # <3>
+    - name: num_concurrent
+      value: "1"
+    - name: max_retries
+      value: "3"
+    - name: tokenized_requests
+      value: "false"
+    - name: tokenizer
+      value: ibm-granite/granite-7b-instruct
+    - name: verify_certificate
+      value: "False"  # <4>
+  pod:
+    container:
+      env:
+        - name: OPENAI_API_KEY  # <5>
+          valueFrom:
+            secretKeyRef:
+              name: lmeval-sa-token
+              key: token
+----
+<1> Use `local-completions` for OpenAI-compatible API endpoints
+<2> Model arguments configure the evaluation client
+<3> HTTPS endpoint of the OAuth-protected InferenceService
+<4> Disable SSL verification for self-signed certificates
+<5> Service account token injected as API key environment variable
+
+Apply the LMEvalJob:
+
+[source,bash]
+----
+kubectl apply -f lmeval-job.yaml -n $NAMESPACE
+----
+
+== Configuration Reference
+
+=== Required Model Arguments
+
+[cols="1,2,1"]
+|===
+|Argument |Description |Example
+
+|`model`
+|Model name for the evaluation
+|`granite`
+
+|`base_url`
+|HTTPS URL of the OAuth-protected InferenceService
+|`$ROUTE/v1/completions`
+
+|`verify_certificate`
+|Set to `"False"` for self-signed certificates
+|`"False"`
+
+|`tokenizer`
+|Tokenizer compatible with the model
+|`ibm-granite/granite-7b-instruct`
+|===
+
+=== OAuth Proxy Endpoints
+
+OAuth-protected InferenceServices typically expose:
+
+* **HTTPS Port**: `8443` (OAuth proxy)
+* **Health Check**: `/health`
+* **API Endpoint**: `/v1/completions`
+* **OAuth Callback**: `/oauth/callback`
+
+== Troubleshooting
+
+=== Common Issues
+
+[cols="1,2,2"]
+|===
+|Problem |Causes |Solution
+
+|OAuth Redirect Loop +
+*(302 redirects to OAuth authorisation endpoint)*
+a|* Missing RBAC permissions
+* Invalid service account token
+* Incorrect OAuth proxy configuration
+a|* Verify RBAC permissions with `kubectl auth can-i`
+* Check service account token validity
+* Ensure OAuth proxy allows programmatic access
+
+|SSL Certificate Errors +
+*(SSL verification failures)*
+|SSL certificate validation issues
+a|* Set `verify_certificate: "False"` in model arguments
+* Use proper CA certificates if available
+* Verify the correct HTTPS endpoint
+
+|Connection Refused +
+*(Connection refused on port 8443)*
+a|* Incorrect service endpoint
+* OAuth proxy not running
+* Network policies blocking access
+a|* Verify InferenceService is running: `kubectl get inferenceservice`
+* Check service endpoints: `kubectl get svc`
+* Test connectivity from within cluster
+|===
+
+=== Debugging Commands
+
+Check RBAC permissions:
+[source,bash]
+----
+kubectl auth can-i get inferenceservices.serving.kserve.io \
+  -n $NAMESPACE \
+  --as=system:serviceaccount:$NAMESPACE:default
+----
+
+Verify service account token:
+[source,bash]
+----
+kubectl get secret lmeval-sa-token -n $NAMESPACE -o jsonpath='{.data.token}' | base64 -d | head -c 50
```

Comment:
```suggestion
kubectl get secret lmeval-sa-token -n $NAMESPACE -o jsonpath='{.data.token}' | base64 -d
```

personally, I would display the token in full, as the next action by the user will likely be to use it with curl/postman?